flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/source/enumerator/DynamoDbStreamsSourceEnumerator.java

```diff
@@ -201,8 +202,21 @@ private SplitGraphInconsistencyTracker trackSplitsAndResolveInconsistencies(
     }
 
     private void assignSplits() {
+        List<DynamoDbStreamsShardSplit> splitsAvailableForAssignment =
+                splitTracker.splitsAvailableForAssignment();
+        assignSplitsInternal(splitsAvailableForAssignment);
+    }
+
+    private void assignSplits(Set<String> finishedSplitIds) {
+        List<DynamoDbStreamsShardSplit> splitsAvailableForAssignment =
+                splitTracker.splitsAvailableForAssignment(finishedSplitIds);
+        assignSplitsInternal(splitsAvailableForAssignment);
+    }
```

Comment:
can we name these methods something like `assignChildSplits()` and `assignAllAvailableSplits()`

Reply:
`assignSplitsInternal` can then be `assignSplits`

Reply:
yeah makes sense